source/vir/src/ast_to_sst.rs

```diff
@@ -246,7 +246,16 @@ impl State {
                         &vars,
                         &body,
                         quant.boxed_params,
-                    )?;
+                    );
+                    let trigs = match trigs {
```

Comment:
This should be removed.  The bug was due to expand-errors being processed during ast-to-sst translation rather than after ast-to-sst translation.  This has been fixed in main, so this workaround is no longer necessary.